shared/k8s/debug.go

```diff
@@ -16,9 +16,14 @@ import (
 )
 
 //nolint:gosec // G101 Not a credential
-const privateKeysSecretResourceName = "validators-private-keys"
-const kubernetesServiceAccountNamespaceFile = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
-const defaultNamespace = "default"
+const (
+	privateKeysSecretResourceNameValidators   = "validators-private-keys"
+	privateKeysSecretResourceNameServicers    = "servicers-private-keys"
+	privateKeysSecretResourceNameFisherman    = "fisherman-private-keys"
```

Comment:
The name of the secret on LocalNet is `fishermen-private-keys`. 

```suggestion
	privateKeysSecretResourceNameFishermen    = "fishermen-private-keys"
```